File: game-server/src/main/java/work/fking/pangya/game/packet/handler/LeaveLobbyPacketHandler.java
package work.fking.pangya.game.packet.handler;

import io.netty.buffer.ByteBuf;
import work.fking.pangya.game.GameServer;
import work.fking.pangya.game.player.Player;
import work.fking.pangya.game.net.ClientGamePacketHandler;
import work.fking.pangya.game.packet.outbound.LobbyResponses;

public class LeaveLobbyPacketHandler implements ClientGamePacketHandler {

    @Override
    public void handle(GameServer server, Player player, ByteBuf packet) {
        player.channel().writeAndFlush(LobbyResponses.ackLeave());
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/handler/LeaveLobbyPacketHandler.kt
package work.fking.pangya.game.packet.handler

import io.netty.buffer.ByteBuf
import work.fking.pangya.game.GameServer
import work.fking.pangya.game.net.ClientPacketHandler
import work.fking.pangya.game.packet.outbound.LobbyResponses
import work.fking.pangya.game.player.Player

class LeaveLobbyPacketHandler : ClientPacketHandler {
    override fun handle(server: GameServer, player: Player, packet: ByteBuf) {
        player.channel().writeAndFlush(LobbyResponses.ackLeave())
    }
}
